### ISSUE TITLE
First star not visible anymore in first two graphs

### ISSUE THREAD

**Issue by SuperTux88** [jeroenheijmans/advent-of-code-charts#89]
I just noticed, that people who only finished one star aren't visible anymore in the first two graphs. I think I didn't notice because the update was only after AoC, so most people were already done, so I didn't notice it right away.

So the new version now only shows the second stars (at the time of the second star), but people who only finished one star for a day just don't have anything in the graph for that day. But I'm also interested in the progress of people who didn't solve both parts yet

And it also means you can't see anymore who took a lot longer for part 2 in the graph. For easy part 2 you usually had this stair pattern, where p1 and p2 were above each others, while on other days where p2 was much harder, you maybe only had the second star for that day later, ending with a more angled line. I liked to see that in the graphs.

**Comment by jeroenheijmans** [comment]
Thanks for the feedback! I don't fully grasp the details yet until I've investigated, but on the surface that sounds like a (regression) bug from recent changes to me. Something to try and look into in fall, should we indeed be blessed with a 2023 edition of AoC.

**Comment by jeroenheijmans** [comment]
I understand now, in the test data when debugging locally the "Default" dataset has a fake user "Private Johnson" who has completed only star 1 of Day 7 which is visible when hovering his entry in the medals overview, but indeed in the first two graphs day 7 doesn't show anything.

I thought for a second about if there's a quick fix (as I have to do a patch release anyways for something else), but changes for this issue here are more impactful unfortunately. So it'll have to wait a bit more.
